### PR TITLE
Add rules 1.14, 1.20, 1.21 and 1.22

### PR DESCRIPTION
**Pull request by stefansjfw**

### CODE REVIEW

File: rules/CIS-1.21_Do_not_setup_access_keys_initially.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  also generates unnecessary management work in auditing and rotating these keys.
+severity: high
+enabled: true
+sql: >
+  SELECT arn
+  FROM public.awsiamcredentialsreport
+  WHERE configuration->>'access_key_1_last_used_date' IS NULL
+    OR configuration->>'access_key_2_last_used_date' IS NULL
+    OR configuration->>'access_key_1_last_used_date' = 'N/A'
+    OR configuration->>'access_key_2_last_used_date' = 'N/A';
```
Comment (PrzemyslawTusinski) [inline rules/CIS-1.21_Do_not_setup_access_keys_initially.yaml:19]: Those can be N/A if access key isn't active

Comment (stefansjfw) [reply rules/CIS-1.21_Do_not_setup_access_keys_initially.yaml]: Added check whether access key is active

File: rules/CIS-1.22_No_policies_with_full_administrative_privileges.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  LEFT JOIN
+    (SELECT arn AS arn1,
+            jsonb_array_elements((policies.supplementaryconfiguration->'attachedPolicies'->>'policyDocument')::JSONB->'Statement') AS statement1
+     FROM awsiampolicy policies
+     WHERE jsonb_typeof((policies.supplementaryconfiguration->'attachedPolicies'->>'policyDocument')::JSONB->'Statement') = 'array' ) multiple_statement_policies ON policies.arn = multiple_statement_policies.arn1
+  LEFT JOIN
+    (SELECT arn AS arn2,
+            (policies.supplementaryconfiguration->'attachedPolicies'->>'policyDocument')::JSONB->'Statement' AS statement2
+     FROM awsiampolicy policies
+     WHERE jsonb_typeof((policies.supplementaryconfiguration->'attachedPolicies'->>'policyDocument')::JSONB->'Statement') = 'object' ) single_statement_policies ON policies.arn = single_statement_policies.arn2
```
Comment (PrzemyslawTusinski) [inline rules/CIS-1.22_No_policies_with_full_administrative_privileges.yaml:27]: What is the scenario when this is object?

Comment (stefansjfw) [reply rules/CIS-1.22_No_policies_with_full_administrative_privileges.yaml:27]: Same object as array element when it is array. There are examples like that in database after full discovery

**Review by PrzemyslawTusinski — APPROVED** [review]

**Pull request merged**